editor.planx.uk/src/@planx/components/Question/Public/Question.tsx

```diff
@@ -24,6 +26,45 @@ export enum QuestionLayout {
 }
 
 const QuestionComponent: React.FC<Question> = (props) => {
+  const [flow, autoAnswerableOptions] = useStore((state) => [
+    state.flow,
+    state.autoAnswerableOptions,
+  ]);
+
+  // Questions without edges function like 'sticky notes' in the graph for editors only
+  let edges: Edges | undefined;
+  if (props.id) edges = flow[props.id]?.edges;
+  if (!edges || edges.length === 0) {
+    return <AutoAnsweredQuestion {...props} answerIds={undefined} />;
+  }
```

Comment:
Currently, Question type nodes without edges do _not_ leave breadcrumbs - but I'm saying here that they should - does this have any unintended consequences / do we need to think about filtering them out downstream now?

Eg Review page (auto answers hidden already); Result page ("reasons"); payload "responses" ?

Reply:
No unintended consequences found in end-to-end services testing :ok_hand: